.github/workflows/continuous-integration.yml

```diff
@@ -1,8 +1,10 @@
 name: "Continuous Integration"
 
 on:
-  - workflow_dispatch
-  - pull_request
+  workflow_dispatch:
+  pull_request:
+    branches:
+      - master
```

Comment:
Why only run on master?

Reply:
@maxakropolis This means the CI workflow will only run on pull requests against the master base branch.